### DISCUSSION TITLE
HyperPay WalletConnect

### DISCUSSION THREAD

**Discussion by MacOMNI** [WalletConnect/walletconnect-monorepo#1262] [Developer Support]
Hello,

We have submitted our wallet "HyperPay" in late Jul and It has been in PENDING REVIEW for a long time.
Could someone review the submission?

Thank you.

**Comment by looter** [comment]
Hi MacOMNI, thanks for your message! It looks like we were having trouble verifying the WalletConnect integration, thanks for the video link! We'll go ahead and approve it now but we suggest a couple of things:
1.  Update the submission after approval and remove the WalletConnect Video Link in the description section
2. Have a direct link for iOS and Android URLs that go to the respective app stores
3. Include a deep link and universal link for the mobile and desktop linking for a better user experience
